src/server/index.tsx

```diff
@@ -148,10 +148,9 @@ app.get('*', async (req: express.Request, res: express.Response) => {
     modals: DEFAULT_MODALS_STATE,
```

Comment:
All looks good. Would be perfect with some tests.

Reply:
Yeah, I mean in this PR we introduced a new concept of right links. Adding tests that verify the right behavior when a navigation item on the right links is clicked is missing in this PR.